CHANGES.rst

```diff
@@ -1,10 +1,147 @@
 .. _gammapy_0p17_release:
 
-0.17 (Unreleased)
------------------
+0.17 (April 1, 2020)
+--------------------
 
-Planned for March 18th 2020
+Summary
++++++++
+
+- Released April 1, 2020
+- 8 contributors
+- 81 pull requests (not all listed below)
+
+**What's new**
+
+Gammapy v0.17 comes with new important features, an improved sub-package
+stucture and a more uniform API.
+
+The main feature introduces in Gammapy v0.17 is event sampling. Based
+on the newly introduced `MapDatasetEventSampler` class, event lists can be
+sampled from a `MapDataset` object. The use of this class is shown in a dedicated
+`event sampling tutorial <notebooks/event_sampling.html>`__. Gammapy v0.17 now
+supports simulation and fitting of temporal models. Both is demonstrated in the
+`lightcurve simulation tutorial <notebooks/light_curve_simulation.html>`__.
+A more general introduction to modelling and fitting in Gammapy is now available
+as a `modelling and fitting tutorial <notebooks/modeling.html>`__
+
+Following the proposal in `PIG 19`_ the sub-package structure of Gammapy was
+unified. Instead of grouping the main functionality by use-case it is now
+grouped by abstractions and data levels. For this all `Dataset` classes
+were moved to the newly introduced `gammapy.datasets`. All `Maker` classes
+for data reduction from the DL3 to the DL4 data level were moved to the new
+`gammapy.makers` sub-package and all high-level `Estimator` classes were moved
+to the new `gammapy.estimators`. In addition a `gammapy.visualization` module
+was introduced to group plotting related functionality into a single namespace.
+The `gammapy.cube`, `gammapy.spectrum` and `gammapy.detect` modules were removed.
```

Comment:
For the latter, the `LiMaMapEstimator` has been removed and replaced with a more general `ExcessMapEstimator` that can also compute errors and upper limit maps. The `TSMapEstimator` now uses the `MapDataset` PSF automatically.

Reply:
Done

Reply:
I think the `LIMaMapEstimator` was never part of a release, I'm not mentioning in...